experimental/packages/exporter-logs-otlp-proto/README.md

```diff
@@ -0,0 +1,110 @@
+# An OTLP exporter to send logs using protobuf over HTTP
+
+[![NPM Published Version][npm-img]][npm-url]
+[![Apache License][license-image]][license-image]
+
+**Note: This is an experimental package under active development. New releases may include breaking changes.**
+
+This module provides exporter for logs in node/web to be used with OTLP (`http/protobuf`) compatible receivers.
+Compatible with [opentelemetry-collector][opentelemetry-collector-url] versions `>=0.32 <=0.50`.
```

Comment:
To make it consistent with the other new exporters, where we only list the proto version.
```suggestion
This module provides an exporter for OTLP (http/json) logs using protocol version `v0.18`.
```